tsl/src/continuous_aggs/create.c

```diff
@@ -1155,14 +1162,17 @@ cagg_validate_query(const Query *query, const bool finalized, const char *cagg_s
 	CAggTimebucketInfo bucket_info, bucket_info_parent;
 	Cache *hcache;
 	Hypertable *ht = NULL, *ht_parent = NULL;
-	RangeTblRef *rtref = NULL;
-	RangeTblEntry *rte;
-	List *fromList;
+	RangeTblRef *rtref = NULL, *rtref2 = NULL;
+	RangeTblEntry *rte = NULL, *rte2 = NULL;
+	JoinType jointype = JOIN_FULL;
+	OpExpr *op = NULL;
+	List *fromList = NIL;
 	StringInfo hint = makeStringInfo();
 	StringInfo detail = makeStringInfo();
 	bool is_nested = false;
 	Query *prev_query = NULL;
 	ContinuousAgg *cagg_parent = NULL;
+	Oid normal_table_id = InvalidOid;
```

Comment:
Do we need this?

Reply:
Just to ensure rte is the one corresponding to the hypertable one.